assets/js/components/settings/settings-module.js

```diff
@@ -220,20 +221,20 @@ class SettingsModule extends Component {
 			<Fragment>
 				{ active ? (
 					<div
-						className={ `
+						className={ classnames( `
 							googlesitekit-settings-module
 							googlesitekit-settings-module--${ slug }
-							googlesitekit-settings-module--active
-							${ error && editActive && isEditing[ moduleKey ] ? 'googlesitekit-settings-module--error' : '' }
-						` }
+							googlesitekit-settings-module--active`,
+						{ 'googlesitekit-settings-module--error': error && editActive && isEditing[ moduleKey ] }
```

Comment:
Missing a tab here :)
```suggestion
							{ 'googlesitekit-settings-module--error': error && editActive && isEditing[ moduleKey ] }
```

Reply:
if we do separate lines using template literal it breaks linting. And `--fix` as above. check another example below.
![image](https://user-images.githubusercontent.com/1387263/74204336-09f19a00-4c73-11ea-9396-670975962f0e.png)

I could do simple strings individually so that 
``` 
(
'googlesitekit-settings-module',
`googlesitekit-settings-module--${ slug }`
...
)
```

or make first 3 classes a single line? and apply same pattern everywhere else too

Reply:
Probably best to start the object on its own line, but have each key/value pair on a newline:

```js
{
  'name': conditional,
  'another-name': conditionalTwo,
}
```

That should work; if not the linting is off and we should adjust it, because as-is it's a bit odd 😄